### PR TITLE
PWN-9210 - striga. enrich usdc

### PR DESCRIPTION
**Pull request by eduardmaximovich**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-9210

## Description of Work

- also fixed [PWN-9193](https://p2pvalidator.atlassian.net/browse/PWN-9193)
- added padding to otp title

[PWN-9193]: https://p2pvalidator.atlassian.net/browse/PWN-9193?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/home/ui/main/HomeInteractor.kt [hunk 1]
```diff
@@ -44,14 +43,16 @@ class HomeInteractor(
...
             }
         }
     }
 
-    suspend fun loadStrigaFiatAccountDetails(): Result<StrigaFiatAccountDetails> {
-        return strigaWalletInteractor.loadFiatAccountAndUserWallet()
+    suspend fun loadDetailsForStrigaAccounts(): Result<Unit> = kotlin.runCatching {
+        strigaWalletInteractor.getFiatAccountDetails()
+        strigaWalletInteractor.getCryptoAccountDetails()
+        Unit
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/home/ui/main/HomeInteractor.kt:55]: try org.p2p.core.utils.StdExtensionsKt#invokeAndForget

Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/home/ui/main/HomeInteractor.kt:55]: return value is not used

File: app/src/main/java/org/p2p/wallet/striga/ui/TopUpWalletPresenter.kt [hunk 2]
```diff
@@ -88,7 +83,7 @@ class TopUpWalletPresenter(
         }
     }
 
-    private suspend fun ensureNeededStrigaDataLoaded() {
+    private suspend fun ensureNeededStrigaDataLoaded(destination: StrigaUserStatusDestination) {
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/striga/ui/TopUpWalletPresenter.kt:86]: im not sure that we can get correct destination if data is not loaded

File: app/src/main/java/org/p2p/wallet/striga/wallet/interactor/StrigaWalletInteractor.kt [hunk 3]
```diff
@@ -34,6 +33,15 @@ class StrigaWalletInteractor(
         return walletRepository.getFiatAccountDetails(eurAccountId).unwrap()
     }
 
+    @Throws(StrigaUSDCAccountNotFound::class, StrigaDataLayerError::class, Throwable::class)
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/striga/wallet/interactor/StrigaWalletInteractor.kt:36]:
```suggestion
    @Throws(StrigaUsdcAccountNotFound::class, StrigaDataLayerError::class, Throwable::class)
```

File: app/src/main/java/org/p2p/wallet/striga/wallet/interactor/StrigaWalletInteractor.kt [hunk 4]
```diff
@@ -34,6 +33,15 @@ class StrigaWalletInteractor(
...
+    suspend fun getCryptoAccountDetails(): StrigaCryptoAccountDetails {
+        val usdcAccountId = walletRepository.getUserWallet()
+            .map { it.usdcAccount?.accountId }
+            .unwrap()
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/striga/wallet/interactor/StrigaWalletInteractor.kt:40]: !!!
successOrNull

**Review by gslevinkov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/striga/ui/TopUpWalletPresenter.kt (hunk 2 above)
Comment (eduardmaximovich) [reply app/src/main/java/org/p2p/wallet/striga/ui/TopUpWalletPresenter.kt:86]: It is, you're right. I missed that data was loading before getting a destination.

File: app/src/main/java/org/p2p/wallet/striga/wallet/interactor/StrigaWalletInteractor.kt (hunk 4 above)
Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/striga/wallet/interactor/StrigaWalletInteractor.kt]: ?: will never be invoked

File: app/src/main/java/org/p2p/wallet/home/ui/main/HomeInteractor.kt (hunk 1 above)
Comment (eduardmaximovich) [reply app/src/main/java/org/p2p/wallet/home/ui/main/HomeInteractor.kt]: the `Result` actually used in the HomePresenter

File: app/src/main/java/org/p2p/wallet/striga/wallet/interactor/StrigaWalletInteractor.kt (hunk 4 above)
Comment (eduardmaximovich) [reply app/src/main/java/org/p2p/wallet/striga/wallet/interactor/StrigaWalletInteractor.kt]: @gslevinkov fixed for both methods

**Review by 1sakov — APPROVED** [review]

**Review by kabramovp2p — APPROVED** [review]

**Pull request merged**